.github/workflows/android-release-to-org-openfoodfacts-scanner.yml

```diff
@@ -88,14 +88,13 @@ jobs:
          SIGN_KEY_ALIAS: ${{ secrets.SIGN_KEY_ALIAS }}
          SIGN_KEY_PASSWORD: ${{ secrets.SIGN_KEY_PASSWORD }}
 
- #     - name: Release AAB
- #       uses: maierj/fastlane-action@v2.2.1
- #       with:
- #         lane: closed_beta
- #         subdirectory: packages/smooth_app/android
- #       env:
- #        SIGN_STORE_PATH: ./../fastlane/envfiles/keystore.jks
- #        SIGN_STORE_PASSWORD: ${{ secrets.SIGN_STORE_PASSWORD }}
- #        SIGN_KEY_ALIAS: ${{ secrets.SIGN_KEY_ALIAS }}
- #        SIGN_KEY_PASSWORD: ${{ secrets.SIGN_KEY_PASSWORD }}
-
+      - name: Release AAB
+        uses: maierj/fastlane-action@v2.2.1
+        with:
+          lane: closed_beta
```

Comment:
is fastlane actually sending to closed_beta ? or internal ?

Reply:
Internal